notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(len(species))
</pre><p>I'd also recommend plotting a bar chart of the count per species. There might be skew between the different classes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>